docs/conf.py

```diff
@@ -107,7 +107,7 @@
     "base_url": "https://minium.com.au/mdanalysis-sphinx-theme/",
     "repo_url": "https://github.com/lilyminium/mdanalysis-sphinx-theme/",
     "repo_name": "mdanalysis-sphinx-theme",
-
+    "mdaofficial": True,
```

Comment:
Just stylistically -- would you be ok with an underscore in between (e.g. `mda_official`, or `mdanalysis_official`) to fit with the other options in the theme?

Reply:
Absolutely, I'm going to make a lot more changes tomorrow